packages/paste-core/components/combobox/src/styles/ComboboxListboxOption.tsx

```diff
@@ -22,7 +25,17 @@ const VariantStyles: {[key in ComboboxListboxOptionProps['variant']]: BoxStylePr
 };
 
 const ComboboxListboxOption = React.forwardRef<HTMLLIElement, ComboboxListboxOptionProps>(
-  ({children, highlighted, variant = 'default', ...props}, ref) => {
+  ({children, highlighted, variant = 'default', virtualItem, ...props}, ref) => {
+    const virtualItemProps = virtualItem
+      ? {
+          position: 'absolute' as PositionOptions,
+          top: 0,
+          left: 0,
+          width: '100%',
+          height: virtualItem.size,
```

Comment:
One thing I was having trouble with was this height. Were you able to get it to calculate correctly for items that run into two lines?

Reply:
Couldnt get it working so only enabling virtualization iff no optionTemplate and no groupItemsBy props